liferay-headless/liferay-portal
# LPS-189893 Publishing apps on startup

https://liferay.atlassian.net/browse/LPS-189893.

**How to test**
- Create an API Builder App and publish it.
- Restart the server.
- The app should be published automatically.

I don't know how to test this automatically.

Rebased on top of https://github.com/brianchandotcom/liferay-portal/pull/137461 and https://github.com/brianchandotcom/liferay-portal/pull/137486. Blocked those are merged
